dev/CommonStyles/CalendarView_themeresources_v1.xaml

```diff
@@ -107,6 +107,8 @@
 
     <Style TargetType="CalendarView" BasedOn="{StaticResource CalendarViewRevealStyle}" />
 
+    <Style x:Key="DefaultCalendarViewStyle" TargetType="CalendarView" BasedOn="{StaticResource CalendarViewRevealStyle}" />
```

Comment:
We should not make change to _v1 resource.

Reply:
See PR https://github.com/microsoft/microsoft-ui-xaml/pull/4620.  We had to do the same thing for ListViewItemPresenter and GridViewItemPresenter.  They define DefaultLiustViewItemStyle and DefaultGridViewItemStyle respectively.